GPU/GPUTracking/TPCClusterFinder/ChargeMapFiller.cxx

```diff
@@ -12,14 +12,15 @@
 /// \author Felix Weiglhofer
 
 #include "ChargeMapFiller.h"
+#include "ChargePos.h"
 #include "Array2D.h"
 
 using namespace GPUCA_NAMESPACE::gpu;
 using namespace GPUCA_NAMESPACE::gpu::deprecated;
 
 GPUd() void ChargeMapFiller::fillChargeMapImpl(int nBlocks, int nThreads, int iBlock, int iThread, GPUTPCClusterFinderKernels::GPUTPCSharedMemory& smem,
                                                GPUglobalref() const Digit* digits,
-                                               GPUglobalref() PackedCharge* chargeMap,
```

Comment:
In principal, you can remove all instances of GPUglobalref() and also GPUsharedref() in here.
They were just needed for the old OpenCL C code.

Reply:
Just to make this more clear: you may only remove GPUglobalref() and GPUsharedref(), the other GPU...ref() must remain. Also, you can only remove them inside the TPCClusterFinder. Part of the other code is still compiler with OpenCL C and needs these macros.